examples/10_multiple_turbine_types.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2021 NREL
+
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+# See https://floris.readthedocs.io for documentation
+
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+from floris.tools import FlorisInterface
+from floris.tools.visualization import visualize_cut_plane
+
+"""
+This example uses an input file where multiple turbine types are defined.
+The first two turbines are the NREL 5MW, and the third turbine is the IEA 10MW.
+"""
+
+# Initialize FLORIS with the given input file via FlorisInterface.
+# For basic usage, FlorisInterface provides a simplified and expressive
+# entry point to the simulation routines.
+fi = FlorisInterface("inputs/gch_multiple_turbine_types.yaml")
+
+# Using the FlorisInterface functions for generating plots, run FLORIS
+# and extract 2D planes of data.
+horizontal_plane = fi.get_hor_plane(x_resolution=200, y_resolution=100, height=90)
+y_plane = fi.get_y_plane(x_resolution=200, z_resolution=100)
+cross_plane = fi.get_cross_plane(y_resolution=100, z_resolution=100)
```

Comment:
Initially, I wasn't sure where this cross-plane was located. I see that the default option is 5 * 126 meters downstream. Perhaps this default option is unnecessary (even potentially confusing) and can just become a required user input? Also, it would be nice to show both the wake behind a 5MW turbine, and behind the 10MW turbine, to clearly show that the wake is significantly larger with the larger turbine. Something like:

```
...
cross_plane1 = fi.get_cross_plane(downstream_dist=50, y_resolution=100, z_resolution=100)
cross_plane2 = fi.get_cross_plane(downstream_dist=5*126 + 10, y_resolution=100, z_resolution=100)
cross_plane3 = fi.get_cross_plane(downstream_dist=10*126 + 10, y_resolution=100, z_resolution=100)

# Create the plots
fig, ax_list = plt.subplots(5, 1, figsize=(10, 8))
ax_list = ax_list.flatten()
visualize_cut_plane(horizontal_plane, ax=ax_list[0], title="Horizontal")
visualize_cut_plane(y_plane, ax=ax_list[1], title="Streamwise profile")
visualize_cut_plane(cross_plane1, ax=ax_list[2], title="Spanwise profile")
visualize_cut_plane(cross_plane2, ax=ax_list[3], title="Spanwise profile")
visualize_cut_plane(cross_plane3, ax=ax_list[4], title="Spanwise profile")
plt.tight_layout()
plt.show()
```